core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -815,14 +815,12 @@ class ZkUtils(zkClientWrap: ZooKeeperClientWrapper,
       None
   }
 
-  def getPartitionsBeingReassigned(): Map[TopicAndPartition, ReassignedPartitionsContext] = {
+  def getPartitionsBeingReassigned(): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
Not sure if there is any other project depending on this method. Perhaps a safer way is to leave this method as it is and let KAFK-5647 change the usage of this method to the one in KafkaZkClient.

Reply:
I had changed it because I didn't want to pass `null` as the second argument to the constructor of `ReassignedPartitionsContext` given that `null` is not a valid value any longer. I changed it back for now though as I assume we will deprecate `ZkUtils` in 1.1.0 and hopefully remove it in 2.0.0 (it is an internal class, but semi widely used).